core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -203,7 +203,10 @@ class AuthorizerIntegrationTest extends AbstractAuthorizerIntegrationTest {
           .transactionStates.asScala.find(_.transactionalId == transactionalId).get
           .errorCode
       )
-    })
+    }),
+    ApiKeys.CONSUMER_GROUP_HEARTBEAT -> ((resp: ConsumerGroupHeartbeatResponse) => Errors.forCode(resp.data.errorCode)),
+    ApiKeys.CONSUMER_GROUP_DESCRIBE -> ((resp: ConsumerGroupDescribeResponse) => Errors.forCode(
+      resp.data.groups.get(0).errorCode))
```

Comment:
For DescribeGroup API, we use the following:
```
    ApiKeys.DESCRIBE_GROUPS -> ((resp: DescribeGroupsResponse) => {
      Errors.forCode(resp.data.groups.asScala.find(g => group == g.groupId).head.errorCode)
    }),
```
Do we need to do the same instead of always getting from index 0?